tests/e2e/scripts/dts-export.ts

```diff
@@ -1,4 +1,4 @@
-const { exportData } = require('../utils/dts-export.ts');
+import { exportData } from '../utils/dts-export';
 
 // TODO: make an actual yargs command and pass common options to exportData so it's easier to build the test data
```

Comment:
no idea what this comment implies, but do you have any thoughts about it?

Reply:
I think it implies that we could use `yargs` to make the arguments passed to the command a little more explicit. ie `node ./dts-export.ts` --file=my-file. Since we only have 1 argument for both scripts I'm not sure it's necessary right, but probably is a good DX improvement so I'm ok with leaving the comment. 

See how we use yargs for the yarn test:e2e command for example:

https://github.com/strapi/strapi/blob/c4a7837bc375b8e2fed4a3d2677d9f11115c348a/tests/scripts/run-e2e-tests.js#L67-L95